packages/chat-stateful-client/src/EventSubscriber.ts

```diff
@@ -62,7 +62,14 @@ export class EventSubscriber {
   };
 
   private onChatMessageReceived = (event: ChatMessageReceivedEvent): void => {
+    // Today we are avoiding how to render these messages. In the future we can
+    // remove this condition and handle this message appropriately.
+    if (event.type.startsWith('RichText/')) {
```

Comment:
Any strong reason for avoiding these?
We can probably convert these to react components and render them.

Reply:
There could be any number of control messages. RichText/Media_CallRecording just happens to be one of them. It is better to filter on what we are expecting (Text or RichText/Html)